### PR TITLE
Update RPi toolchain installation guide

### PR DESCRIPTION
**Pull request by SalimTerryLi**
Update the correct GCC version and aarch64 toolchain.

### CODE REVIEW

**Comment by hamishwillee** [comment]
@bkueng @dagar What @SalimTerryLi  has done here looks great to me. Can you confirm that the versions of compilers indicated etc are what we expect?

**Comment by SalimTerryLi** [comment]
All of those toolchains(armhf and aarch64) won't work on Ubuntu 20.04 host machine, as it depends on libc6-2.29, which is not available on RPi now(still 2.27). Anyone who want to build for RPi must use older Ubuntu 18.04 or docker build. A lot of documentation work.

They can pass the build and provide binary files, but is extreme new so cannot run on actual Pi. The really working one is documented separately in user docs.

**Comment by hamishwillee** [comment]
@SalimTerryLi 

Re https://github.com/PX4/PX4-Devguide/pull/1103#issuecomment-721486508 I added a Warning at the beginning of the RasPi section which just says which Ubuntu version to use. Does that fix this consideration?

If so, do you think this is safe/good to merge?

**Comment by SalimTerryLi** [comment]
Excellent! Thanks for your supplement.
It's generally right in my view, and is OK to merge.

File: en/setup/dev_env_linux_ubuntu.md [hunk 1]
```diff
@@ -63,6 +63,11 @@ sudo add-apt-repository --remove ppa:team-gcc-arm-embedded/ppa
 
 <!-- NOTE: RaPi docker toolchain (for comparison) here: https://github.com/PX4/containers/blob/master/docker/Dockerfile_armhf -->
 
+> **Warning** Use Ubuntu **18.04** or [Docker](https://github.com/PX4/containers/blob/master/docker/Dockerfile_armhf) for building RasPi!
+  *Do not use Ubuntu 20.04*. The *armhf* or *aarch64* toolchains on a Ubuntu 20.04 host machine depend on *libc6-2.29* which is not available on RPi at time of writing (it has *libc6-2.27*).
+
+They can pass the build and provide binary files, but is extreme new so cannot run on actual Pi. The really working one is documented separately in user docs.
```
Comment (bkueng) [inline en/setup/dev_env_linux_ubuntu.md]: Add link to the user doc?

**Review by bkueng — APPROVED** [review]

File: en/setup/dev_env_linux_ubuntu.md (hunk 1 above)
Comment (SalimTerryLi) [reply en/setup/dev_env_linux_ubuntu.md]: [This section](https://dev.px4.io/master/en/setup/building_px4.html#other-boards) is a little disorganized. Should all of those boards locate somewhere else?(for example, user docs).
I think only this rough link should be provided here: https://docs.px4.io/master/en/flight_controller/

Comment (hamishwillee) [reply en/setup/dev_env_linux_ubuntu.md]: @SalimTerryLi 
1. "The really working one is documented separately in user docs." like @bkueng says, where is that link to go to?
2. The information should be in one place. I would be happy for this section you point to go into the user docs. What we would leave here is a link to that. 

We should fix this now and do that other work as a separate action. OK?

**Comment by SalimTerryLi** [comment]
Modified. @hamishwillee

**Comment by hamishwillee** [comment]
Thanks @SalimTerryLi ! I did a minor subedit and will now merge.

Note that the PX4 user guide and Devguide will shortly merge. At that point we might look at moving to having just ONE source of build information :-)

**Pull request merged**